For inspection

VolodymyrSud opened the pull request.

VolodymyrSud committed:
Creation:
- Main class
- method main + implementation
- Menu class
- method run + implementation
- private static constants:
ENCODE_FILE
DECODE_FILE
EXIT_NUMBER
MENU_INFO
- methods:
itemToEncodeFile
itemToDecodeFile
itemToExit

VolodymyrSud committed:
Creation:
- private static final constants:
OPTIONS_FOR_ENCODE
OPTIONS_FOR_DECODE
MOVE_RIGHT_ON_3
MOVE_RIGHT_ON_4
MOVE_RIGHT_ON_5
MOVE_LEFT_ON_3
MOVE_LEFT_ON_4
MOVE_LEFT_ON_5
BRUTE_FORCE

- private static boolean isRunning
- final CeasarCypher
- final Scanner

- implementation to itemToEncodeFile
- implementation to itemToDecodeFile
- implementation to itemToExit

- folder Keys
- class Key
- method readeFile + implementation
- protected method shiftRight + implementation
- protected method shiftLeft + implementation

- class CeasarCypher
- private static final constant ALPHABET
- public method encode + implementation
- public method decode + implementation

- class FilePathNotValid
- public method FilePathNotValid + implementation
- private method validateFilePath + implementation
- private
method createTargetFilePath + implementation

- classes:
EncodingWithShiftBy3Letters
EncodingWithShiftBy4Letters
EncodingWithShiftBy5Letters
DecodingWithShiftBy3Letters
DecodingWithShiftBy4Letters
DecodingWithShiftBy5Letters
BruteForce

VolodymyrSud committed:
Creation:
overridden readeFile method in pre-created classes:
EncodingWithShiftBy3Letters
EncodingWithShiftBy4Letters
EncodingWithShiftBy5Letters
DecodingWithShiftBy3Letters
DecodingWithShiftBy4Letters
DecodingWithShiftBy5Letters
BruteForce

VolodymyrSud committed:
12 Unversioned Files

Volodymyr95 commented on src/Keys/BruteForce.java:
```diff
@@ -0,0 +1,32 @@
+                        shiftLeft(3);
+                        return fileText.trim().replace("ввв", "в");
+                    }
+                    if (substring.equals("гггг")) {
```
гггг make it variable or constant

Volodymyr95 commented on src/Keys/BruteForce.java:
```diff
@@ -0,0 +1,32 @@
+                        shiftLeft(4);
+                        return fileText.trim().replace("гггг", "г");
+                    }
+                    if (substring.equals("ґґґґґ")) {
```
make variable or constant

VolodymyrSud committed:
Changes:
- Created a constant LAST_POINT.
- rename cipherText and plainText to textBuilder.

VolodymyrSud committed:
Delete .gitignore

File is not required

VolodymyrSud committed:
Delete encodings.xml

File is not required

VolodymyrSud committed:
Delete misc.xml

File is not required

VolodymyrSud committed:
Delete modules.xml

File is not required

VolodymyrSud committed:
Delete uiDesigner.xml

File is not required

VolodymyrSud committed:
Delete index.pb

File is not required

VolodymyrSud committed:
Delete index.pb

File is not required

VolodymyrSud committed:
Delete vcs.xml

File is not required

VolodymyrSud committed:
Replaced 4 and 5 with SHIFT_ON_4 and SHIFT_ON_5
